website/docs/components/color-picker/color-picker-code.md

```diff
@@ -2,4 +2,38 @@
 title: Example
 ---
 
-@## Примеры color-picker
+@## ColorPicker
+
+Widget allows the user to select the color of another component from the pre-defined palette or specify a custom color via its hexadecimal value.
+
+@example defaultExtended
+
+@## PaletteManager
+
+Using PaletteManager you can add custom colors by entering color in HEX format in input and after this clicking on check icon inside input or pressing Enter on keyboard. In addition, you can remove custom colors by clicking on close icon on each item.
```

Comment:
As mentioned in [Wikipedia](https://en.wikipedia.org/wiki/Web_colors) HEX format is better known as a "hexadecimal format" or a "hex triplet"